Mathlib/CategoryTheory/Category/KleisliCat.lean

```diff
@@ -52,7 +52,7 @@ instance KleisliCat.category {m} [Monad.{u, v} m] [LawfulMonad m] : Category (Kl
   -- refine' { id_comp' := _, comp_id' := _, assoc' := _ } <;> intros <;> ext <;> unfold_projs <;>
   --  simp only [(· >=> ·), functor_norm]
   refine' { id_comp := _, comp_id := _, assoc := _ } <;> intros <;> refine funext (fun x => ?_) <;>
-  simp [CategoryStruct.id, CategoryStruct.comp, (· >=> ·)]
+  simp (config := { unfoldPartialApp := true }) [CategoryStruct.id, CategoryStruct.comp, (· >=> ·)]
```

Comment:
Can you please mention this simp config in the commit message?

Reply:
Done.